ccip/config/evm/Avalanche_Mainnet.toml

```diff
@@ -18,4 +18,6 @@ PriceDefault = '1 gwei'
 BlockHistorySize = 24
 
 [HeadTracker]
-PersistenceEnabled = false
```

Comment:
`PersistenceEnabled = false` allows HeadTracker to stop persisting heads. This operation generates a significant load on the database, but it does not provide any benefits on chains with fast finality. 
We should not override the flag.

Reply:
Hmm, this is interesting. I've just backported the state from the CCIP repo. Exactly how it is right now on the develop branch